action.yaml

```
@@ -12,7 +12,7 @@ inputs:
     required: true
   transform-configuration: 
     description: 'Transform app configurations based on build configuration'
-    required: true
+    required: false
     default: true
```

should this be modified?
```suggestion
    default: false
```

---

No it needs to default to true otherwise it will break all builds aside @tracplus-hpaterson 's

---

Negative, sorry. My description was the error here, rather than the code.

Most of our legacy style repos use the `App.config` XML which does require the transform, so it should default to true. I'm updating the PR descrption now.